src/sentry/utils/snuba.py

```diff
@@ -713,7 +757,6 @@ def _prepare_query_params(query_params):
             "groupby": query_params.groupby,
             "conditions": query_params.conditions,
             "aggregations": query_params.aggregations,
-            "project": project_ids,
```

Comment:
intentional ?

Reply:
Yea. If the dataset is `events` or `transactions`, we update the `query_params.kwargs` with a `project`. If the dataset is `outcomes`, we update it with `organization` [above](https://github.com/getsentry/sentry/pull/15022/files#diff-001cb5ccc9f5ad59f8745e09d9db2051R721).